lib/hammer_cli_foreman_tasks/recurring_logic.rb

```diff
@@ -27,6 +27,30 @@ class CancelCommand < HammerCLIForeman::DeleteCommand
       build_options
     end
 
+    class ClearCommand < HammerCLIForeman::DeleteCommand
+      option '--cancelled', :flag, _("Only delete cancelled recurring logics")
+      option '--finished', :flag, _("Only delete finished recurring logics")
+      def request_params
+        params = super
+        raise ArgumentError, "Please specify if you want to remove cancelled or finished recurring logics using --cancelled or --finished " unless (options['option_cancelled'] || options["option_finished"])
+        cancelled = "state=cancelled" if options["option_cancelled"]
+        finished = "state=finished" if options["option_finished"]
+        params["search"] = cancelled || finished
```

Comment:
So what happens if the user provides both `--cancelled` and `--finished`?

Reply:
I think it might be unclear if ` --cancelled and --finished` mean cancelled **and** finished or cancelled **or** finished.
So this will only clear cancelled.
Should using both flags just remove cancelled **or** finished?

Reply:
Either it should remove RLs which are cancelled *or* finished, or it should raise an error. Silently ignoring one of the flags doesn't feel right

Reply:
I think raising an error is a better solution. Added it